go/internal/hashing/xxh3_memo_table.go

```diff
@@ -166,16 +168,13 @@ func (s *BinaryMemoTable) Size() int {
 }
 
 // helper function to easily return a byte slice for any given value
-// regardless of the type if it's a []byte, parquet.ByteArray,
-// parquet.FixedLenByteArray or string.
+// regardless of the type.
```

Comment:
should probably clarify that this only works for `[]byte`, `string` or something which fulfills the `ByteSlice` interface